apps/style/netsim/style.scss

```diff
@@ -473,12 +472,57 @@ svg {
     }
 
     .send_widget_footer {
-      text-align: right;
       margin: 10px;
+      overflow: auto;
+
+      .right-side-controls {
+        float: right;
+        text-align: right;
+      }
+
+
     }
   }
 
-  .netsim_tabs {
+  // Packet size slider and labels, which appear in the
+  // send widget footer.
+  .netsim_packet_size_control {
+    label {
+      display: inline-block;
+    }
+
+    .slider-inline-wrap {
+      display: inline-block;
+      margin-top: 3.5px;
+      width: 300px;
+
+      .slider-labels {
+        margin-bottom: 1px;
+
+        .max-value {
```

Comment:
Nesting should be no greater than 4, but was 5<br>Selector should have depth of applicability no greater than 2, but was 5